caveclient/chunkedgraph.py

```diff
@@ -1082,6 +1113,7 @@ def get_original_roots(self, root_id, timestamp_past=None) -> np.ndarray:
         """
         root_id = root_id_int_list_check(root_id, make_unique=True)
 
+        # TODO feels like there should be a timestamp call here if timestamp is set?
```

Comment:
how should get_original_roots behave when version is set? currently the timestamp doesnt passed into get_root_timestamps

Reply:
I think this should stay as is. This is a call where the timestamp can be ignored, since it works only on lineage graph ordering of root ids. We assume people are using the root ids they want to use, so fixing timestamps shouldn't matter.